frontend/gateway/gateway.go

```diff
@@ -187,18 +216,26 @@ func (gf *gatewayFrontend) Solve(ctx context.Context, llbBridge frontend.Fronten
 		ReadonlyRootFS: readonly,
 	}, rootFS, lbf.Stdin, lbf.Stdout, os.Stderr)
 
+	if lbf.err != nil {
```

Comment:
What happens if both `lbf.err != nil and err != nil` ?

Reply:
we will ignore the err. err is likely just "exit code 1" but `lbf.err` is hopefully the reason for that exit code.